src/Conversation.tsx

```diff
@@ -53,11 +54,18 @@ const ConversationComponent = ({
 
       {isRequesting && (
         <div className={styles.assistantOutputContainer}>
+          <img
+            className={styles.conversationChatIcon}
+            src={hardDiamondLogo}
+            alt="Hard AI logo"
+          />
           <div className={styles.assistantContainer}>
             {progressText.length ? (
-              <Content content={progressText} role="assistant" />
+              <div className={styles.progressStream}>
+                <Content content={progressText} role="assistant" />
+              </div>
             ) : null}
-            {<Content content={assistantStream} role="assistant" />}
```

Comment:
Maybe this was a remnant of `assistantStream.length ? (...`

Reply:
This is interesting, essentially always rendering. Maybe the check got moved into the Content component directly?

Reply:
Yeah - `Content` renders an empty div if the sanitizedContent is an empty string